2009-05-04  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Volumes.pm: Ignore another line of output from parted
		(GPT-related warning). Thanks Andreas Schuldei for noticing this.
Index: trunk/lib/setup-storage/Volumes.pm
===================================================================
--- trunk.orig/lib/setup-storage/Volumes.pm
+++ trunk/lib/setup-storage/Volumes.pm	
@@ -146,7 +146,8 @@
 
       # now we test line by line - some of them may be ignored
       next if ($line =~ /^Disk / || $line =~ /^Model: / || $line =~ /^\s*$/
-        || $line =~ /^WARNING: You are not superuser/);
+        || $line =~ /^WARNING: You are not superuser/
+        || $line =~ /^Warning: Not all of the space available to/);
 
       # determine the logical sector size
       if ($line =~ /^Sector size \(logical\/physical\): (\d+)B\/\d+B$/) {
